color_helper.sublime-settings

```diff
@@ -394,7 +394,7 @@
             "base_scopes": ["text.ass"],
             "scanning": ["constant.other.color"],
             "color_class": "ass_abgr",
-            "color_trigger": "(?i)&h|\\\\[1-4]?c"
+            "color_trigger": "(?:&H|(?<=\\\\[1-4]c)|(?<=\\\\c))[0-9a-fA-F]"
```

Comment:
`(?<=` lookbehind only supports fixed-width expression.

Reply:
yep, I've had to use the double lookbehind before.